drivers/base/bootvid/i386/pc/pc.h

```diff
@@ -19,6 +19,22 @@ extern UCHAR PixelMask[8];
 #define __outpw(Port, Value) \
     WRITE_PORT_USHORT((PUSHORT)(VgaRegisterBase + (Port)), (USHORT)(Value))
 
+FORCEINLINE
+VOID
+SetPaletteEntry(IN ULONG Id,
+                IN ULONG Rgb)
```

Comment:
It's better to pass `PALETTE_ENTRY` to `SetPaletteEntry` function.

Reply:
There were two functions, one of them accepted `PALETTE_ENTRY`, but the other one worked with XRGB quad as is. If we want to pass `PALETTE_ENTRY` here, it leads to unnecessary code duplication. Also we can avoid code duplication by modifying function `InitPaletteWithTable`, but it will just overcomplicate things.

Reply:
Let's keep InitPaletteWithTable() as it is.